frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Web Terminal
+    As a user, I should be able to use web terminal
+
+
+Background:
+    Given user is at developer perspecitve
+    And user has installed Web Terminal operator
+    And user has created Web Terminal instance
+    And user has selected namespace "aut-terminal-basic"
+
+
+@regression, @smoke
+Scenario: Web Terminal window
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    Then user will see the terminal window
+
+
+@regression
+Scenario: Web Terminal in new tab
+    Given user is at topology page
+    When user clicks on the Web Terminal icon on the Masthead
+    And user clicks on Open Terminal in new tab button
```

Comment:
I think one more step should be there- `And user sees the terminal window`

Reply:
I haven't added a separate statement but updated one statement.

Reply:
For automating the scenario we need to wait for the Terminal window to open right?